components/tabs/tab-mixin.js

```diff
@@ -0,0 +1,219 @@
+import '../colors/colors.js';
+import { css, html, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { getFocusPseudoClass } from '../../helpers/focus.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+const keyCodes = {
+	ENTER: 13,
+	SPACE: 32
+};
+
+export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
+	static get properties() {
+		return {
+			ariaSelected: { type: String, reflect: true, attribute: 'aria-selected' },
+			// eslint-disable-next-line lit/no-native-attributes
+			role: { type: String, reflect: true },
+			selected: { type: Boolean, reflect: true },
+			tabIndex: { type: Number, reflect: true },
```

Comment:
Does this need to be a reactive property? The reason I ask is that `tabIndex` is already a native JavaScript property for an HTML element, and this component doesn't need to re-render itself if it changes. `d2l-tab-internal` also doesn't define it as a reactive property for those reasons.

Reply:
Was for the same reason as ariaSelected - mainly for external transparency but I'm good to remove from this hook